.github/workflows/ci.yml

```
@@ -0,0 +1,30 @@
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '23'
+
+      - name: Check Java version
+        run: java -version
+
+      - name: Build and test with Maven
+        run: mvn clean test
+        env:
+          JAVA_HOME: ${{ steps.setup-java.outputs.java-home }}
```

actionlint (1.7.10)

[error] property "java-home" is not defined in object type {cache-hit: string; distribution: string; path: string; version: string}

(expression)